doc/user/content/sql/create-sink.md

```diff
@@ -103,7 +120,7 @@ they occur. To only see results after the sink is created, specify `WITHOUT SNAP
 
 ## Detail
 
-- Materialize currently only supports the following sinks:
+- Materialize currently only supports the following [sink formats](#format):
```

Comment:
The rest of this paragraph still calls these "sinks", not "sink formats". I think "sink" might still be the better term but I'm easy to convince otherwise.

Reply:
I think "sink formats" makes sense here in context.